_posts/2022-10-19-arrow-flight-sql-jdbc.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: "Expanding Arrow's Reach with a JDBC Driver for Arrow Flight SQL"
+date: "2022-10-19 09:00:00 -0500"
+author: pmc
+categories: [application]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+We're excited to announce that as of version 10.0.0, the Arrow project
+now includes a [JDBC][jdbc] driver [implementation][impl] based on
+[Arrow Flight SQL][flight-sql].  This is courtesy of a software grant
+from [Dremio][dremio-arrow], a data lakehouse platform. Contributors
+from Dremio developed and open-sourced this driver implementation, in
+addition to designing and contributing Flight SQL itself.
+
+Flight SQL is a protocol[^1] for client-server database interactions.
+It defines how a client should talk to a server and execute queries,
+fetch result sets, and so on.  Underneath, it builds on [Arrow Flight
+RPC][flight], a framework for efficient transfer of Arrow data across
+the network.  While Flight RPC is flexible and can be used in any type
+of application, from the beginning, it was designed with an eye
+towards the kinds of use cases that Flight SQL supports.
+
+With this new JDBC driver, applications can talk to any database
+server implementing the Flight SQL protocol using familiar JDBC APIs.
+Underneath, the driver sends queries to the server via Flight SQL and
+adapts the Arrow result set to the JDBC interface, so that the
+database can support JDBC users without implementing additional APIs
+or its own JDBC driver.
+
+[^1]: Despite the name, Flight SQL is **not** a SQL dialect, nor is it
```

Comment:
This seems important and worth reiterating--maybe put it in the paragraph itself instead of a footnote/